docs/ref/models/database-functions.txt

```diff
@@ -288,6 +290,10 @@ Usage example::
 
     .. attribute:: lookup_name = 'week_day'
 
+.. class:: ExtractIsoWeekDay(expression, tzinfo=None, **extra)
+
+    .. attribute:: lookup_name = 'iso_week_day'
+
```

Comment:
I would add a line with description, e.g.
```
.. class:: ExtractIsoWeekDay(expression, tzinfo=None, **extra)

    .. versionadded:: 3.1

    Returns the ISO-8601 week day with day 1 being Monday and day 7 being
    Sunday.

    .. attribute:: lookup_name = 'iso_week_day' 
```
Moreover, it would be good to add it to examples for `DateTimeField` and `DateField`.